agent/src/beerocks/slave/backhaul_manager/backhaul_manager_thread.cpp

```diff
@@ -2146,15 +2156,7 @@ bool backhaul_manager::handle_client_capability_query(ieee1905_1::CmduMessageRx
     //Check if it is an error scenario - if the STA specified in the Client Capability Query message is not associated
     //with any of the BSS operated by the Multi-AP Agent [ though the TLV does contain a BSSID, the specification
     // says that we should answer if the client is associated with any BSS on this agent.]
-    bool associated_client_found = false;
-    for (const auto &slave : slaves_sockets) {
-        auto associated_clients_map = slave->associated_clients_map;
-        if (get_sta_bssid(associated_clients_map, client_info_tlv_r->client_mac()) !=
-                network_utils::ZERO_MAC) {
-            associated_client_found = true;
-            break;
-        }
-    }
+    bool associated_client_found = (bool)get_sta_radio(client_info_tlv_r->client_mac());
```

Comment:
Please no C-style casts. I believe this works:
```suggestion
    bool associated_client_found { get_sta_radio(client_info_tlv_r->client_mac()) };
```